### PR TITLE
Typo fix in Russian

### PR DESCRIPTION
**Pull request by Gotoro**

### CODE REVIEW

File: src/main/resources/i18n/messages_ru.properties [hunk 1]
```diff
@@ -656,7 +656,7 @@ ranked1v1.queuePopTimer=Игра через {0,number,0}\:{1,number,00}
 game.create.generatedMap=Сгенерировать карту
 game.avgRating.format={0,number,\#}
 game.mapGeneration.notification.title=Карта генерируется...
-game.mapGeneration.notification.message=Ожадайте. (Version\: {0}, Seed\: {1})
+game.mapGeneration.notification.message=Ожидайте. (Version\: {0}, Seed\: {1})
```
Comment (norraxx) [inline src/main/resources/i18n/messages_ru.properties:659]: :+1:

**Comment by 1-alex98** [comment]
No all edits need to be done in poeeditor!!!!

**Comment by 1-alex98** [comment]
https://poeditor.com/join/project/2oWfdaZQZj

**Comment by 1-alex98** [comment]
Tell me your email so I can accept you. alexander.von.trostorff@gmail.com

**Pull request closed without merging**